pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -296,3 +296,33 @@ func TestCephObjectStoreUserController(t *testing.T) {
 	assert.Equal(t, "Ready", objectUser.Status.Phase, objectUser)
 	logger.Info("PHASE 5 DONE")
 }
+
+func TestBuildStatusInfo(t *testing.T) {
```

Comment:
This does not test BuildStatusInfo but the generation of the user secret name. It's tested throughout `TestBuildUpdateStatusInfo` so you don't need this test.

Reply:
Done.